### PR TITLE
fix(tests): update model unit tests for new batch_dim type

### PR DESCRIPTION
**Pull request by sauyon**
We no longer support non-tuple batch_dim.

### CODE REVIEW

File: tests/unit/_internal/models/test_model.py [hunk 1]
```diff
@@ -59,7 +59,9 @@
     batchable: false
   classify:
     batchable: true
-    batch_dim: 0
+    batch_dim:
```
Comment (bojiang) [inline tests/unit/_internal/models/test_model.py:62]: shall we have
```
input_batch_dim: 0
output_batch_dim: 0
```
here? @parano

Comment (parano) [reply tests/unit/_internal/models/test_model.py:62]: yes, I thought you had some refactoring work for batch_dim that will do this?

Comment (bojiang) [reply tests/unit/_internal/models/test_model.py:62]: @parano yeah but I need an extra approval since it's simplified a lot.

Comment (bojiang) [reply tests/unit/_internal/models/test_model.py:62]: will do that in a small refactoring PR

**Review by bojiang — APPROVED** [review]

**Pull request merged**